WebDriverAgentLib/Categories/XCUIApplication+FBHelpers.m

```diff
@@ -45,7 +43,12 @@ - (BOOL)fb_deactivateWithDuration:(NSTimeInterval)duration error:(NSError **)err
 - (NSDictionary *)fb_tree
```

Comment:
fb_tree is only used to get the json source, but the xml one is handled in other place